agent/agent.go

```diff
@@ -606,7 +606,8 @@ func (a *Agent) runTask(ctx context.Context, tcInput *taskContext, nt *apimodels
 	})
 
 	tskCtx = utility.ContextWithAttributes(tskCtx, tc.taskConfig.TaskAttributes())
-	tskCtx, span := a.tracer.Start(tskCtx, fmt.Sprintf("task: '%s'", tc.taskConfig.Task.DisplayName))
+	tskCtx, span := a.tracer.Start(tskCtx, "task")
+	span.SetAttributes(attribute.String("task.id", tc.task.ID))
```

Comment:
There should be a task ID attribute set on the span already (`evergreen.task.id`).
Is it missing from this span?

Reply:
Ah, true, removed.